aten/src/ATen/native/native_functions.yaml

```diff
@@ -2701,16 +2701,29 @@
 
 - func: matrix_exp_backward(Tensor self, Tensor grad) -> Tensor
 
+# DEPRECATED: Use torch.minmax instead
 - func: _aminmax(Tensor self) -> (Tensor, Tensor)
   variants: function
   dispatch:
     CPU, CUDA: _aminmax_all
 
+# DEPRECATED: Use torch.minmax instead
 - func: _aminmax.dim(Tensor self, int dim, bool keepdim=False) -> (Tensor, Tensor)
   variants: function
   dispatch:
     CPU, CUDA: _aminmax
 
+- func: minmax(Tensor self, *, int? dim=None, bool keepdim=False) -> (Tensor min, Tensor max)
```

Comment:
Why minmax and not aminmax?
Is there a numpy sister function that we match here?

cc @mruberry

Reply:
No, numpy (and python array API) don't have minmax.

Reply:
And is this function behavior the same as the existing `min`/`max` or existing `amin`/`amax`?

Reply:
From offline discussion with @mruberry, I will update this to torch.aminmax.